crates/re_types/definitions/rerun/attributes.fbs

```diff
@@ -17,3 +17,10 @@ attribute "attr.rerun.component_recommended";
 ///
 /// Only applies to the fields of an archetype.
 attribute "attr.rerun.component_optional";
+
+/// Override the type of a field.
+///
+/// The only permitted value is "half", which can only be used to override the type of a ushort.
```

Comment:
is "f16" or "float16" perhaps more clear than "half"?

Reply:
"half" matches the terms that the way that fbs files use for their types: int, short, long, float, double.  That doesn't necessarily make it any more clear though I'm happy to use float16 instead.